sandbox/Controls.qml

```diff
@@ -188,4 +188,14 @@ GridLayout {
         to: 100
         value: 40
     }
+
+    StatusBanner {
+        id: banner
+        width: 360
+        topPadding: 20
+        backgroundColor: Theme.palette.primaryColor3
+        bordersColor: Theme.palette.primaryColor2
+        fontColor: Theme.palette.primaryColor1
```

Comment:
So you see here how you're forced to set `backgroundColor`, `bordersColor` and `fontColor`?

I don't want to worry about that as a user. I just wanna say: "Render a warning banner" or "render a success banner".

So accomplish this it'd be better of `StatusBanner` had a `type` which is an enum to allow consumers of this API to choose what kind of banner they want:

```qml
StatusBanner {
  text: "This is a warning banner"
  type: StatusBanner.Type.Danger
}

StatusBanner {
  text: "This is a success banner"
  type: StatusBanner.Type.Success
}

StatusBanner {
  text: "This is a default banner"
}
```

The default could use those `primaryColor`s you've used here.

Reply:
There's a bunch of components in StatusQ that do this kind of thing. Here's one example to give you some inspiration: https://github.com/status-im/StatusQ/blob/master/src/StatusQ/Components/StatusExpandableItem.qml#L35-L39

Reply:
Yes, got it! Great idea! I'm working on that